api/test/integration/projects/projects-key-costs.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpStatus } from '@nestjs/common';
+import { projectsContract } from '@shared/contracts/projects.contract';
+import { TestManager } from 'api/test/utils/test-manager';
+
+describe('Projects key costs', () => {
+  let testManager: TestManager;
+  let jwtToken: string;
+
+  beforeAll(async () => {
+    testManager = await TestManager.createTestManager();
+    const result = await testManager.setUpTestUser();
+    jwtToken = result.jwtToken;
```

Comment:
Good copy paste, but auth is not required for this feature, please remove it (it might cause confusion in the future)